chia/full_node/mempool.py

```diff
@@ -389,6 +398,12 @@ def create_bundle_from_mempool_items(
             if not item_inclusion_filter(name):
                 continue
             try:
+                await eligible_coin_spends.process_fast_forward_spends(
```

Comment:
if this throws, that means we should drop `item` from the mempool, right?
or maybe we don't do that here

Reply:
Yes, we do so when we can't proceed with the fast forward.

Reply:
That was the plan, but I didn't realize when I updated this that the mempool change landed into main already.